test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,46 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const fixedPort = 8080;
```

Comment:
> I thought to use it because of your comment on this: #12639

@sebastianplesciuc Not sure which comment you mean.

> Also as I understood from the common.PORT in parallel tests issue, it was planned to use a linting rule against using common.PORT in parallel tests.

If/when that happens, `eslint-disable` comments can be used for any remaining valid `common.PORT` uses in `parallel`. Changing them now to accommodate a rule that may never come to pass is probably putting the cart before the horse.

Regardless, none of that applies if the test is moved to `sequential`. :-D

Lastly: I hope none of this is too frustrating for you. I appreciate all the work you're doing and I know it's not fun to get contradictory suggestions from people.